### DISCUSSION TITLE
Improve documentation

### DISCUSSION THREAD

**Discussion by niranjanghule** [spockframework/spock#1451] [General]
Hi,

I was reading through section "How Do I Get Started?" on [homepage](https://spockframework.org/). The section mentions to read 10 reasons for using spock but doesn't provide any link/reference.
I checked the documentation but couldn't find it.
It would help reader if you attach a link or remove it if not required.

**Comment by szpak** [comment]
I don't know "10 reasons..." neighter, but one can always read my own [7 reasons](https://speakerdeck.com/szpak/7-reasons-why-bother-learning-spock) ;-) (unfortunately, slightly outdated in some points, but they could be updated if needed).

**Comment by leonard84** [comment]
It refers to https://github.com/spockframework/spock/wiki/10-Reasons-to-use-Spock
The missing link has been added to the website.
